### PR TITLE
Update intro docs

### PR DESCRIPTION
**Pull request by abemassry**

### CODE REVIEW

**Comment by mparke** [comment]
needs rebase

**Comment by mparke** [comment]
`tohttps://login.alpha.linode.com` needs a space

**Comment by mparke** [comment]
do you mind wrapping urls in paragraphs in `<code></code>` ?

File: docs/src/components/intros/Errors.js [hunk 1]
```diff
@@ -14,8 +14,7 @@ export default function Errors() {
           <code>4xx</code> codes indicate an error on your side, and
           <code>5xx</code> codes indicate an error on our side. An error on your
           side might be an invalid input, a required parameter being omitted, and
-          so on. Errors on our side shouldn't ever happen, and we're probably going to be
-          running around with our hair on fire if you ever see them.
+          so on. Errors on our side shouldn't happen often, if they do please let us know.
```
Comment (mparke) [inline docs/src/components/intros/Errors.js]: wording: `...often. If an error does occur, please let us know.` ?

File: docs/src/components/intros/Filtering.js [hunk 2]
```diff
@@ -132,6 +108,30 @@ export default function Filtering() {
     }'`}
           </code>
         </pre>
+        <p>
+          Below is a list of all available operators. Operators are keys of a Filter JSON object, their value must be of the appropriate
+          type, and they are evaluated as described below.
```
Comment (eatonphil) [inline docs/src/components/intros/Filtering.js]: Could you break this into two sentences? (After the first comma)

File: docs/src/components/intros/Filtering.js [hunk 3]
```diff
@@ -132,6 +108,30 @@ export default function Filtering() {
     }'`}
           </code>
         </pre>
+        <p>
+          Below is a list of all available operators. Operators are keys of a Filter JSON object, their value must be of the appropriate
```
Comment (na3d) [inline docs/src/components/intros/Filtering.js]: Just a minor tweak.
> Operators are keys of a Filter JSON object.  Their value...

**Pull request merged**